docs/src/main/paradox/api-reference/schemas.md

```diff
@@ -1,252 +1,153 @@
 ```
 
-## Get shape
-###**`GET /schemas/{orgId}/{domainId}/{name}/{version}/shapes/{fragment}`**
-**Retrieves** the shape specified in **fragment** from the schema in **name & version** from the domain **domainId** inside the organization **orgId**.
-
-### Request path and query parameters
-| Field         | Type          | Description                                                                                                                   |
-| ------------- |-------------  | ---------------------------------------------                                                                                 |
-| orgId         | String        | The unique identifier for the organization.                                                                                   |    
-| domainId      | String        | The unique identifier for the domain for this particular organization.                                                        |
-| name          | String        | The unique identifier for the schema for this particular domain.                                                              |
-| version       | String        | The unique identifier for the schema version for this particular schema, following the [semantic format](http://semver.org/). |
-| fragment      | String        | The unique short identifier for a shape inside the schema.                                                                    |
-
-
-
-### Response JSON-LD Object
-| Field         | Type          | Description                                   |
-| ------------- |-------------  | --------------------------------------------- |
-| @id           | String        | The unique identifier for the shape.          |
-| rev           | Long          | The current revision of the schema.           |
-| deprecated    | Boolean       | whether the schema is deprecated or not.      |
-| published     | Boolean       | whether the schema is deprecated or not.      |
-| *             | *             | The payload of the shape.                     |
-
-### Status Codes
-* **200 OK** - Request completed successfully.
-* **404 Not Found** - The requested shape does not exist.
-
-### Example request
-```bash
-curl -v "https://bbp-nexus.epfl.ch/{environment}/{version}/schemas/nexus/core/subject/v1.0.0/shapes/AgeValueShape"
+#### Example
+
+Request
+:   @@snip [schema-delete.sh](../assets/api-reference/schemas/schema-delete.sh)
+
+Response
+:   @@snip [schema-ref-new.json](../assets/api-reference/schemas/schema-ref-delete.json)
+
+### Search schemas
+
+Follows the general definition of searching in a collection of resources.
+
 ```
+GET /v0/schemas/{orgId}/{domId}/{name}
+      ?q={full_text_search_query}
+      &filter={filter}
+      &from={from}
+      &size={size}
+      &deprecated={deprecated}
+      &published={published}
+```
+... where 
+
+* `{orgId}` filters the resulting schemas to belong to a specific organization.
```

Comment:
* `{orgId}` the organization the schema belongs to.
* `{domId}` the domain the schema belongs to.
* `{name}` the schema name.